Add new volume rendering tests and baselines

biagas opened the pull request:
### Description

These were created by @ARSanderson during his VTK-9/OSPRay work.


### Type of change

<!-- Please check one of the boxes below -->

~~* [ ] Bug fix~~
~~* [ ] New feature~~
~~* [ ] Documentation update~~
* [X] Other~~ <!-- please explain with a note below -->
Testing

### How Has This Been Tested?

Running the tests on pascal.

### Reminders:

- Please follow the [style guidelines][1] of this project.
- Please perform a self-review of your code before submitting a PR and asking others to review it.
- Please assign reviewers (see [VisIt's PR procedures][2] for more information).

### Checklist:

<!-- For items in this checklist that do not apply, simply insert two tilde chars, `~~`, just ahead of the left bracket char, `[` at the beginning of a line. Each line ends with two tilde chars to make doing such ~~strikeouts~~ easy. -->

- [ ] I have commented my code where applicable.~~
- [ ] I have updated the release notes.~~
- [ ] I have made corresponding changes to the documentation.~~
- [ ] I have added debugging support to my changes.~~
- [ ] I have added tests that prove my fix is effective or that my feature works.~~
- [ ] I have confirmed new and existing unit tests pass locally with my changes.~~
- [ ] I have added new baselines for any new tests to the repo.~~
- [ ] I have NOT made any changes to [*protocol* or *public interfaces*][3] in an RC branch.~~

[1]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/StyleGuide.html
[2]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/pr_create.html#reviewers
[3]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/RCDevelopment.html#communication-protocols-and-public-apis

biagas committed:
Add new volume plot tests and baselines.

These were contributed by @ARSanderson during his VTK-9/OSPRay work.

biagas committed:
Update baselines.

biagas committed:
Merge branch 'develop' into task/biagas/add_new_volume_rendering_tests

biagas commented:
@ARSanderson there are issues with volume_10 and volume_20.
When run in serial or parallel modes the images are blank.
When run in scalable,parallel,icet mode the images are not blank.

Is it expected to get blank images in serail and parallel but not scalable,parallel,icet?

Here's the generated scalable,paralell,icet image for volume_10
![volume_10](https://github.com/visit-dav/visit/assets/17075318/77c04c75-0e8f-4e78-a527-6f7ba4b25df2)

Here's the generated scalabel,parallel,icet image for volume_20
![volume_20](https://github.com/visit-dav/visit/assets/17075318/0680bb1c-9c53-4b8d-913b-5d5aee90761f)

ARSanderson commented:
Yes, in some cases there is an issue with the VTK actor dropping off to lala land.  I have been able to reproduce it but only with certain data sets. Others it works. Quite odd. @brugger1  and I discussed adding a ticket for this issue as IIRC we saw it with the old rendering too. I did not add a ticket because we wanted the dust to settle on the merge/updates etc (in the hopes it would not be an issue further).

biagas committed:
Merge branch 'develop' into task/biagas/add_new_volume_rendering_tests

biagas committed:
Update ospray and volume baselines.
Add volume_10,20,30 tests to skip list for serial and parallel modes.
Skip volume_30 entirely for scalable,parallel,icet mode as it crashes.

biagas committed:
Merge branch 'develop' into task/biagas/add_new_volume_rendering_tests

biagas committed:
Merge branch 'develop' into task/biagas/add_new_volume_rendering_tests
